ui/admin-portal/e2e/protractor.conf.js

```diff
@@ -16,19 +16,19 @@
 
 // Protractor configuration file, see link for more information
 // https://github.com/angular/protractor/blob/master/lib/config.ts
-
 const { SpecReporter } = require('jasmine-spec-reporter');
+const httpMock = require('protractor-http-mock/lib/httpMock');
 
 exports.config = {
   allScriptsTimeout: 11000,
   specs: [
     './src/**/*.e2e-spec.ts'
   ],
   capabilities: {
-    'browserName': 'chrome'
+    'browserName': 'firefox'
```

Comment:
Why not Chrome? I think that most of us use Chrome. Can we support both?
@EhsanEhrari

Reply:
Chrome's frequent updates sometimes lead to compatibility issues with the latest version of ChromeDriver, making it necessary to downgrade Chrome to run tests smoothly. By including Firefox, we ensure flexibility and reliability in our testing setup. While Firefox serves as the default browser due to its stability and consistent compatibility, we can certainly support both browsers to accommodate various testing scenarios effectively.

So, in summary:
- If Firefox is available, Protractor will use it as the default browser.
- If Firefox is not available, Protractor will attempt to use another browser specified in the configuration, such as Chrome, as a fallback option.

What do you think?

Reply:
Hi @EhsanEhrari, and if both browsers are available, can we support both?

Reply:
Hi @sadatmalik ,

Protractor does support running tests in multiple browsers simultaneously. However, based on my testing experience, I've found that Chrome tends to execute the tests much faster compared to Firefox. Therefore, I will go with the Chrome to ensure tests run quicker and faster. Additionally, if there are specific scenarios where testing in Firefox is necessary or desired, we can certainly configure Protractor to support both browsers as needed.